nav2_system_tests/src/behaviors/spin/spin_behavior_tester.hpp

```diff
@@ -57,7 +57,9 @@ class SpinBehaviorTester
   // Runs a single test with given target yaw
   bool defaultSpinBehaviorTest(
     float target_yaw,
-    double tolerance = 0.1);
+    double tolerance = 0.1,
+    bool wait_action = true,
```

Comment:
Can we rename this to `nonblocking_action` to be more clear for myself when I read this in a few years :laughing: